src/style.css

```diff
@@ -1 +1,78 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+body {
+  margin: 0;
+  padding: 0;
+  font-weight: 500;
+  font-family: Roboto, sans-serif;
+  background-color: rgb(235, 235, 235);
+
+  height: 5000px;
+}
+
+.header {
+  position: fixed;
+  width: 100vw;
+  background-color: #fff;
+}
+
+.container {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  max-width: 1100px;
+  margin: 0 auto;
+  padding: 0 50px;
+}
+
+.header__logo {
+  height: 40px;
+  width: 40px;
+}
+
+.header__logo_img {
+  height: 100%;
+  width: 100%;
+}
+
+.nav-list {
+  display: flex;
+  justify-content: space-around;
+  margin: 22px;
+  padding: 0;
+}
+
+.nav-list__item {
+  position: relative;
+  margin-left: 20px;
```

Comment:
you don't need margin-left for first element

Reply:
Thanks, I remove margin from first child ;)